lib/sentry/logger_backend.ex

```diff
@@ -93,7 +93,8 @@ defmodule Sentry.LoggerBackend do
 
     if Logger.compare_levels(level, state.level) != :lt and
          not LoggerUtils.excluded_domain?(meta[:domain] || [], state.excluded_domains) do
-      log(level, msg, meta, state)
+      :ok = log(level, msg, meta, state)
+      :ok
```

Comment:
I think this should rather be

```suggestion
      _ = log(level, msg, meta, state)
      :ok
```

as we're not guaranteed that `log/4` always returns `:ok`, right?

Reply:
That's correct, and it's essentially what I wanted your feedback on. I'll change it as you suggest.